tests/conftest.py

```diff
@@ -57,27 +90,42 @@ def session_id():
 
 
 @pytest.yield_fixture(scope='session')
-def kafka_server(request, docker, docker_ip_address, unused_port, session_id):
+def kafka_server(request, docker, docker_ip_address,
+                 unused_port, session_id, ssl_folder):
     image = request.config.getoption('--docker-image')
-    docker.pull(image)
+    # Don't pull image if it available localy. Helps when developing new docker
+    # builds
+    if not docker.images(image):
+        docker.pull(image)
```

Comment:
I have some pygo/kafka:2.11_0.9.0.1 image locally (pulled on previous `make test` in master).
If you (or somebody else) changes this image on docker hub, I'll never pull it (with this code).

For testing new docker builds you can use --docker-image option in pytest.